### ISSUE TITLE
ntfy button is local URL instead of public url

### ISSUE THREAD

**Issue by mabed-fr** [0x2142/frigate-notify#79]
Hello,

Thank for the work !

I have find a "problem"

In ntfy notification this button is local URL

![image](https://github.com/0x2142/frigate-notify/assets/25350431/ab2903b2-c3d7-4cc3-af71-774b27eafd34)

Is this normal behavior?

```yaml
frigate:
  # Frigate host URL (ex. https://frigate.yourdomain.tld)
  # This is required for both collection methods
  server: http://192.168.1.254:5000
  # Set to true if using SSL & a self-signed certificate
  ignoressl: true
  # Public / internet-facing Frigate URL, if different from above server address
  public_url: https://nvr**********.test.fr
  ```

  Regards

**Comment by freefd** [comment]
Dear @mabed-fr, to facilitate the investigation, please share your frigate-notify configuration.

> [!CAUTION]
> Do not forget to obscure sensitive things like public IP addresses, domains, credentials, etc.

**Comment by 0x2142** [comment]
Hi @mabed-fr - Thanks for reporting this. Looks like when I added the [action button](https://github.com/0x2142/frigate-notify/blob/f16d11d2efd99e1493386d5c90ff40ee30844423/notifier/nfty.go#L38), I am only using the frigate server URL - and forgot to check if `public_url` is configured.

Good catch! And thanks again for reporting this. I'll get it resolved 👍

**Comment by mabed-fr** [comment]
in any case the action button is great!

**Comment by 0x2142** [comment]
Fixed in [v0.3.1](https://github.com/0x2142/frigate-notify/releases/tag/v0.3.1)
